src/split_cams.py

```diff
@@ -83,20 +84,20 @@ def compute_homography(kp1, kp2, matches):
             - inliers (int): The number of inliers found during RANSAC.
     """
     # Extract the matched keypoints
-    src_pts = np.float32([kp1[m.queryIdx].pt for m in matches]).reshape(-1, 1, 2)
-    dst_pts = np.float32([kp2[m.trainIdx].pt for m in matches]).reshape(-1, 1, 2)
+    src_pts = np.float32([kp1[m.queryIdx].pt for m in matches]).reshape(-1, 1, 2)  # type: ignore
```

Comment:
I didn't succeed to fix this cleanly @MateoLostanlen , if you want to take a look to avoid ignoring the errors